languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,30 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QUELLES_GROUPE_NOMINAL" name="quelle ou quelles groupe nominal" default="temp_off">
+      <rule>
+          <pattern>
+              <marker>
+                  <token regexp="yes">qu(&apostrophe;)?</token>
+                  <token>elle</token>
+              </marker>
+              <token postag="N.*|J.*" postag_regexp="yes"/>
+          </pattern>
+          <message>Une erreur semble être détectée dans cette phrase. Souhaitez-vous écrire <suggestion>quelle</suggestion> ?</message>
+          <example correction="quelle">Il m'a raconté <marker>qu'elle</marker> belle journée il avait passé.</example>
+      </rule>
```

Comment:
FPs for this first rule in the RE:

- Elle s'aperçoit **qu'elle** seule peut les voir
- Elle a son propre show **qu'elle** coécrit sur la BBC

Please check the rule editor once more and adapt the rule accordingly.